server/db/models/course.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../connection');
+const sequelize = require('sequilize');
+
+module.exports = Course;
+
+class Course extends sequelize.Model {}
+Course.init(
+  {
+    id: {
+      type: sequelize.UUID,
+      defaultValue: sequelize.UUIDV4,
+      unique: true,
+      primaryKey: true,
+    },
+    name: {
+      type: sequelize.STRING(1024),
```

Comment:
Any particular reason for these string length choices?

Reply:
Yes. The default string length is 255 characters, and I made my choices based on what I think may be the longest course name possible. Where 255 is quite possibly not long enough. 

I may be wrong. Its a bit of a safety measure though. Thats the only reason.

Reply:
Who knows. Maybe I'm overthinking a trivial thing. :man_shrugging:

Reply:
Sounds good to me! Thanks for explaining.